astropy/modeling/fitting.py

```diff
@@ -1156,6 +1154,62 @@ def _convert_input(x, y, z=None, n_models=1, model_set_axis=0):
     return farg
 
 
+class MinimizeFitter(Fitter):
+    """
+
+    A fitter for the minimize optimizer.
```

Comment:
Could you make it clear this is scipy's minimize optimizer.
A reference to the documentation would be good too.